src/FileServiceManager/FileService/Controllers/FileController.cs

```diff
@@ -0,0 +1,54 @@
+﻿using FileService.DAL;
+using FileService.Models;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace FileService.Controllers
+{
+    [Produces("application/json")]
+    [ApiController]
+    [Route("api/[controller]")]
+    [Authorize]
+    public class FileController : ControllerBase
+    {
+        private readonly IStorage _storage;
+
+        public FileController(IStorage storage)
+        {
+            _storage = storage;
+        }
+
+        [HttpGet("Download/{fileId}")]
+        public async Task<FileMetadata> DownLoadFileAsync(Guid fileId, int companyId)
+        {
+            //identity check
```

Comment:
think of this identity check as an action filter, to not have the same logic written in every method

Reply:
I remove the comments. The implementation will follow